### ISSUE TITLE
Remove cooling alerts when thermometer is maxed out

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#100]
From a meeting today. It sounds weird when the thermometer stays completely red, but the model in the background is decreasing, so we still get cooling alerts.

@terracoda do you think that this should be the same for warming alerts? I'm not sure we discussed that at the meeting.

**Comment by terracoda** [comment]
@zepumph, I hear the cooling alert way less now, likely due to the change in the temperature threshold. 

I commented in issue 101 (https://github.com/phetsims/friction/issues/101#issuecomment-418823674) about what alerts I think will be nice.

To answer you question, though, yes, I think it might be a good idea to have an alert that communicates more clearly that things are hot, hotter or very hot, rather than just warmer.

**Comment by zepumph** [comment]
In the above commit, I turned off cooling alerts when hotter than the evaporation limit. I know that this is not the same as the maxed out thermometer, but right now that is easier, and I want to be consistent (because I cut that same corner in #101). Please let me know what you think.

We can adjust to base this off of the actual visual "maximum" of the thermometer if needed.

**Comment by zepumph** [comment]
@terracoda please review.

**Comment by terracoda** [comment]
Sounds great @zepumph! I hear a a still very hot alert when thermometer is maxed out. I will close this issue. If we need to tweaking of wording that will be a new issue.
